astropy/modeling/tests/test_fitters.py

```diff
@@ -403,3 +404,110 @@ def test_param_cov(self):
 
         assert_allclose(fmod.parameters, beta.A.ravel())
         assert_allclose(olscov, fitter.fit_info['param_cov'])
+
+@pytest.mark.skipif('not HAS_SCIPY')
+class Test1DFittingWithOutlierRemoval(object):
+    def setup_class(self):
+        self.x = np.linspace(-5., 5., 200)
+        self.model_params = (3.0, 1.3, 0.8)
+
+        def func(p, x):
+            return p[0]*np.exp(-0.5*(x - p[1])**2/p[2]**2)
+
+        self.y = func(self.model_params, self.x)
+
+    def test_with_fitters_and_sigma_clip(self):
+        from astropy.stats import sigma_clip
```

Comment:
Move this import to the top of the file and write it as `from ...stats import sigma_clip`.